artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -3604,4 +3605,14 @@ private void deployReloadableConfigFromConfiguration() throws Exception {
       return externalComponents;
    }
 
+   @Override
+   public Set<Consumer> queueConsumersQuery(SimpleString queueName) {
```

Comment:
This method isn't necessary. You can use the `locateQueue` method and simply invoke `getConsumers` on the returned value.